src/couchbeam_view.erl

```diff
@@ -406,9 +410,9 @@ parse_view_options([{skip, Value}|Rest], #view_query_args{options=Opts}=Args) ->
 parse_view_options([{list, Value}|Rest], #view_query_args{options=Opts}=Args) ->
     Opts1 = [{list, Value}|Opts],
     parse_view_options(Rest, Args#view_query_args{options=Opts1});
-parse_view_options([{keys, Value}|Rest], Args) ->
```

Comment:
I'm using couchbase/server:community-4.1.0 and it simply don't work with POST, I don't remember specific error message but as far as I remember the POST method was not allowed. Maybe it's version dependent.